assets/app/view/game/player.rb

```diff
@@ -127,6 +127,21 @@ def render_info
             h('td.right', @game.format_currency(@game.bidding_power(@player))),
           ])
         end
+
+        if @game.respond_to?(:debt)
```

Comment:
is this needed? how do other games do this?

Reply:
I only found in 1849 something similiar. There it is just not shown.
I'm not sure if we want to show it always or just if > 0 because its probably happening rarely that a player gets a debt.

Reply:
ok delete it

Reply:
I'm not a fan of having hidden game states, but for now I removed it. We'll have to see how often in happens in a real game.